src/server/controllers/Grbl/GrblController.js

```diff
@@ -107,6 +107,10 @@ class GrblController {
             this.ready = false;
             if (err) {
                 log.error(`Unexpected error while reading/writing serial port "${this.options.port}":`, err);
+                this.emit('electron-error', {
```

Comment:
OK - this is for a few changes but the ticket SPECIFICALLY SAID to log only GRBL alarms and errors, not application alarms and errors.  

You only needed to listen to those firmware messages and handle them, but instead added code for logging ALL application errors which is redundant since we already do this.

So, to clarify, GRBL errors and alarm messages (emitted by runner or feeder/sender) need to be handled in this unique manner, we already have logging in place for application errors.